packages/web-extension/src/walletManager/walletManager.types.ts

```diff
@@ -0,0 +1,44 @@
+import { AsyncKeyAgent } from '@cardano-sdk/key-management';
+import { ObservableWallet, storage } from '@cardano-sdk/wallet';
+
+export interface WalletManagerProps {
+  walletName: string;
+}
+
+export interface WalletManagerActivateProps<P = string, O = unknown> {
```

Comment:
It could also be a number coming from an enum...

Reply:
Yeah, I'm just not sure if unconstrained type will work for a type union. Maybe it will :sweat_smile: Feel free to just mark this change request as resolved, or maybe do a 'string' | 'number'

Reply:
Agree. Updated it in [9f664a4](https://github.com/input-output-hk/cardano-js-sdk/pull/502/commits/9f664a4152ddc7853f8ce6753ba4475f3016187d)